src/solvers/dgmulti/sbp.jl

```diff
@@ -244,6 +244,56 @@ function StartUpDG.RefElemData(element_type::Hex,
     M, Pq, Drst, LIFT)
 end
 
+function StartUpDG.RefElemData(element_type::Hex,
```

Comment:
I can't comment there but the method above has a branch `if D isa AbstractPeriodicDerivativeOperator` that is not needed anymore with this specialization.